.editorconfig

```
@@ -4,7 +4,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Changing this affects the entire project, including the javascript/typescript code
This change, along with adjustment of the formatting, would be its own issue. And we will discuss this during the MMM
```suggestion
indent_size = 2
```